spec/acceptance/should_tolerate_mixed_case_spec.rb

```diff
@@ -134,22 +134,25 @@
         /Registry_value\[HKLM.Software.Vendor.PuppetLabsTest_MixedCase_\w+\\SubKey1\\ValueString3\].ensure: created/,
         /Registry_value\[HKLM.Software.Vendor.PuppetLabsTest_MixedCase_\w+\\SubKey1\\ValueString4\].ensure: created/,
     ]
+
+    agent_is_x64 = false 
     windows_agents.each do |agent|
+      agent_is_x64 = is_x64(agent)
```

Comment:
If the nodeset includes both 32-bit and 64-bit agents, this doesn't seem right... I think you'd want to filter the nodes for x64, then change the `execute_manifest` below to `execute_manifest_on` and only use 64-bit agents.